product_docs/docs/edbcloud/beta/reference/index.mdx

```diff
@@ -0,0 +1,193 @@
+2. [Request the device code](http://localhost:8000/edbcloud/latest/reference/#request-the-device-code).
+3. [Authorize as a user](http://localhost:8000/edbcloud/latest/reference/#authorize-as-a-user).
+4. [Request the token](http://localhost:8000/edbcloud/latest/reference/#request-the-token).
+
+
+
+## Query the Authentication Endpoint
+
+Use the following command to get the information returned by the authentication endpoint that you need later:
+
+```
+curl https://portal.edbcloud.com/api/v1/auth/provider
+```
+    
+The response returns the `clientId`, `issuerUri`, `scope`, and `audience`. For example:
+```
+{
+  "clientId": "pM8PRguGtW9yVnrsvrvpaPyyeS9fVvFh",
+  "issuerUri": "https://edbcloud.us.auth0.com",
+  "scope": "openid profile email offline_access",
+  "audience": "https://portal.edbcloud.com/api"
+}
+```
+
+## Request the Device code
+
+Use the following call to get a device code: 
+```
+   curl --request POST \
+     --url 'https://edbcloud.us.auth0.com/oauth/device/code' \
+     --header 'content-type: application/x-www-form-urlencoded' \
+     --data 'client_id=pM8PRguGtW9yVnrsvrvpaPyyeS9fVvFh' \
+     --data scope='openid profile email offline_access' \
+     --data audience='https://portal.edbcloud.com/api'
+```
+
+The response returns:
+
+- `device_code` - the unique code for the device. When you go to the `verification_uri` in your browser-based device, this code is bound to your session. You use this code in your [request for a token](#request-the-token).
+- `user_code` - the code you input at the `verification_uri` to authorize the device. You use this code when you [authorize yourself as a user](#authorize-as-a-user).
+- `verification_uri` - the URL you use to authorize your device.
+- `verification_uri_complete` - the complete URL you use to authorize the device. You can use this URL to embed the user code in your app's URL.
+- `expires_in` indicates the lifetime (in seconds) of the `device_code` and `user_code`.
+- `interval` indicates the interval (in seconds) at which the app should poll the token URL to request a token.
+
+For example: 
+```
+{
+  "device_code": "KEOY2_5YjuVsRuIrrR-aq5gs",
+  "user_code": "HHHJ-MMSZ",
+  "verification_uri": "https://edbcloud.us.auth0.com/activate",
+  "expires_in": 900,
+  "interval": 5,
+  "verification_uri_complete": "https://edbcloud.us.auth0.com/activate?user_code=HHHJ-MMSZ"
+}
+```
+## Authorize as a User
+
+1. Go to the `verification_uri` in your web browser, enter your `user_code`, and select **Continue**.
+
+4. Select **Confirm** on the Device Confirmation dialog.
+
+5. Select **Continue with Microsoft Azure AD** on the EDB Cloud Welcome screen.
+
+6. Log in with your Azure AD credentials.
+ 
+
+## Request the Token
+
+Use the `curl --request POST` command to request a token. For example, 
+```
+curl --request POST \
+  --url 'https://edbcloud.us.auth0.com/oauth/token' \
```

Comment:
plz change this to: `https://auth.edbcloud.com/oauth/token`, we are going to use customized domain name